ui/src/main/java/com/theoplayer/android/ui/FullscreenHandler.kt

```diff
@@ -35,7 +35,6 @@ internal class FullscreenHandlerImpl(private val view: View) : FullscreenHandler
         val window = activity.window
 
         // Hide system bars
-        WindowCompat.setDecorFitsSystemWindows(window, false)
```

Comment:
Honestly, I don't remember where I got this code from. 😅

It looks like [this guide](https://developer.android.com/develop/ui/views/layout/immersive) is the best resource at the moment, and it doesn't mention `setDecorFitsSystemWindows` at all. So I suppose it's not needed. 👍